### DISCUSSION TITLE
Record/Snap trigger on Person outside of Zone

### DISCUSSION THREAD

**Discussion by Fettkeewl** [blakeblackshear/frigate#3651] [Ask A Question] [answered]
Hi there!

I feel like I have followed the instructions to use zones properly and not get notified when persons walk outside of my property.
But it does not work as intended, as I can't see an error in my setup I wonder if it might be a bug? 
Please see the image
![why trigger](https://user-images.githubusercontent.com/25577040/184622066-c8e006f6-c43a-4c03-9268-e7669002d04e.png)

**Comment by NickM-27** [comment, marked as answer]
Presence in zones is based on the bottom center of the objects bounding box. That zone is right up to the bottom of the sidewalk so I can definitely see how the bottom center of the persons bounding box was inside the zone for a frame or two. 

I'd recommend trimming the frame down some with that in mind.

**Comment by Fettkeewl** [comment]
![image](https://user-images.githubusercontent.com/25577040/184630326-33e4d95d-a2c7-4595-ac6b-6f87a2c092ce.png)
Alright then, ill try it like this, even more removed

**Reply by NickM-27** [reply]
Personally, I would bring in that part on the top left a bit, like I said it is bottom center so no one could be standing on top of the doorway without having been in the zone already, but a person walking by the sidewalk could get confused and bounding box including the wall a bit which could still catch on there.

**Reply by NickM-27** [reply]
Other than that I think it looks good though

**Reply by Fettkeewl** [reply]
Alright I understand what you mean, thanks alot :)! Will fix

> Personally, I would bring in that part on the top left a bit, like I said it is bottom center so no one could be standing on top of the doorway without having been in the zone already, but a person walking by the sidewalk could get confused and bounding box including the wall a bit which could still catch on there.
